endoreg_db/views/report_service_views.py

```
@@ -0,0 +1,346 @@
+    ReportListSerializer
+)
+
+logger = logging.getLogger(__name__)
+
+class ReportListView(APIView):
+    """
+    API-Endpunkt für paginierte Report-Listen mit optionaler Filterung
+    GET /api/reports/
+    """
+    
+    def get(self, request):
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)